op-e2e/inbox/inbox_test.go

```diff
@@ -0,0 +1,154 @@
+package inbox
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	batcherFlags "github.com/ethereum-optimism/optimism/op-batcher/flags"
+	op_e2e "github.com/ethereum-optimism/optimism/op-e2e"
+	"github.com/ethereum-optimism/optimism/op-e2e/bindings"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
+	"github.com/ethereum-optimism/optimism/op-e2e/system/e2esys"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	ctx, _          = context.WithTimeout(context.Background(), 20*time.Second)
+	cost            = big.NewInt(1500000000000000)
+	depositVal      = new(big.Int).Mul(cost, big.NewInt(1000))
+	mockStorageAddr = common.Address{}
+)
+
+func TestBatchInboxFunctionSuccess(t *testing.T) {
+	op_e2e.InitParallel(t)
+
+	sys, l1Client := startSystemWithBatchInboxContract(t)
+	t.Cleanup(sys.Close)
+
+	// Wait for batch submitted and check event
+	requireEventualBatcherTx(t, &sys.Cfg, l1Client, 12*time.Second)
+}
+
+func startSystemWithBatchInboxContract(t *testing.T) (*e2esys.System, *ethclient.Client) {
+	cfg := e2esys.DefaultSystemConfig(t)
+	cfg.DataAvailabilityType = batcherFlags.BlobsType
+	cfg.BatcherTargetNumFrames = eth.MaxBlobsPerBlobTx
+	cfg.DeployConfig.UseInboxContract = true
+	c, ok := cfg.Nodes["sequencer"]
+	require.True(t, ok, "sequencer is required")
+	c.SafeDBPath = t.TempDir()
```

Comment:
Looks like this line is not needed? You can also remove this line from the l2 blob test so that unnecessary code is not introduced again.